### PR TITLE
Integrate the sessions API at admin/sessions route

### LINKED ISSUE #630: Implementation of API for sessions on '/admin/sessions'.
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

adminsessions

### PR DESCRIPTION
**Pull request by sumedh123**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #630 .

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-633-open-event-frontend-fossasia.surge.sh

**Comment by sumedh123** [comment]
@geekyd @CosmicCoder96 @harshitagupta30 @niranjan94 Please review.

**Comment by sumedh123** [comment]
@geekyd I think it's just because the response is not having the dates in it. When I tried it on local orga-server, I created sessions with dates and those worked fine.

File: app/controllers/admin/sessions/list.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    {
+      propertyName : 'event.name',
+      title        : 'Event Name'
+    },
+    {
+      propertyName : 'title',
+      title        : 'Title'
+    },
+    {
+      propertyName : 'speakers',
```
Comment (geekyd) [inline app/controllers/admin/sessions/list.js]: Disable sorting for this column

**Comment by harshitagupta30** [comment]
@geekyd this is happening because the data is null in the server response. I also faced the same issue.

File: app/controllers/admin/sessions/list.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+export default Controller.extend({
+  columns: [
+    {
+      propertyName : 'event.name',
```
Comment (geekyd) [inline app/controllers/admin/sessions/list.js]: Disable sorting for this column, it wont work for now

**Comment by sumedh123** [comment]
@geekyd Updated. Please review.

**Comment by niranjan94** [comment]
Deployment link: https://pr-633-open-event-frontend-fossasia.surge.sh

**Comment by sumedh123** [comment]
@CosmicCoder96 @geekyd @harshitagupta30 Please review.

**Comment by niranjan94** [comment]
Deployment link: https://pr-633-open-event-frontend-fossasia.surge.sh

**Pull request merged**